pkgs/applications/emulators/rpcs3/default.nix

```diff
@@ -70,6 +70,13 @@ gcc11Stdenv.mkDerivation {
     ++ lib.optional alsaSupport alsa-lib
     ++ lib.optional waylandSupport wayland;
 
+  postInstall = ''
+    # Taken from https://wiki.rpcs3.net/index.php?title=Help:Controller_Configuration
+    install -D 99-ds3-controllers.rules $out/etc/udev/rules.d/99-ds3-controllers.rules
+    install -D 99-ds4-controllers.rules $out/etc/udev/rules.d/99-ds4-controllers.rules
+    install -D 99-dualsense-controllers.rules $out/etc/udev/rules.d/99-dualsense-controllers.rules
```

Comment:
Should these be `$out/etc/udev/rules.d/` or `$out/lib/udev/rules.d/`? I'm actually not sure.